opm/simulators/linalg/PreconditionerFactory_impl.hpp

```diff
@@ -162,6 +162,12 @@ struct StandardPreconditioners
         F::addCreator("ILUn", [](const O& op, const P& prm, const std::function<V()>&, std::size_t, const C& comm) {
           return createParILU(op, prm, comm, prm.get<int>("ilulevel", 0));
         });
+        F::addCreator("DuneILU", [](const O& op, const P& prm, const std::function<V()>&, std::size_t, const C& comm) {
+            const double w = prm.get<double>("relaxation", 1.0);
+            const int n = prm.get<int>("ilulevel", 0);
+            const bool resort = prm.get<bool>("resort", false);
+            return wrapBlockPreconditioner<DummyUpdatePreconditioner<Dune::SeqILU<M, V, V>>>(comm, op.getmat(), n, w, resort);
```

Comment:
Dune::SeqILU should not be wrapped in the DummyUpdatePreconditioner, I think it needs an actual update(). I suggest creating a new wrapper class that reconstructs on update(), ideas welcome.

Reply:
> should only the Dune SeqILU use it?

I think so, any preconditioner that stores anything other than a reference to the matrix will need a nontrivial update(). There is another, SeqILDL, that also seems to need this. I am not familiar with that one.